website/docs/guide/directives.md

```diff
@@ -21,9 +21,9 @@ paginate: true
 
 ### Front matter
 
-**YAML front matter** is a syntax often used for keeping metadata of Markdown in a lot of tools (e.g. [Jekyll site generator](https://jekyllrb.com/docs/front-matter/)). Marp has also supported front matter as the container to define directives.
+Like many tools (e.g. [Jekyll site generator](https://jekyllrb.com/docs/front-matter/)), Marp uses **YAML front matter** to store Markdown metadata. In addition, Marp also uses the front matter to define directives.
```

Comment:
Marp is actually using the front-matter syntax only for storing global directives, and there is no opinion how to store Markdown metadata. So I thought it might cause less confusion to avoid making mention of the metadata.